src/cola/wraptext.js

```diff
@@ -10,7 +10,7 @@ export default function wrapText(ctx, text, maxWidth) {
   do {
     let fits = false;
     const m = ctx.measureText(lines[activeLine]);
-    if (m.width > maxWidth) {
+    if (m.width > maxWidth && lines[activeLine].length > 1) {
```

Comment:
Yeah this is probably a good idea. I wanted to build a better truncation. Maybe if it's longer than the maxwidth and it's one word.

Also, you're probably getting this to break a lot because of a font thing. I need to add a script that adds the fonts for your platform.